### PR TITLE
fix: user popover ui fixes

### PR DESCRIPTION
**Pull request by CzarekDryl**
[tw](https://tw.auroracreation.com/app/tasks/15839237)

## Description

- Fixed few ui cases of user popover

Resolves [https://github.com/JoinColony/colonyCDapp/issues/2054](https://github.com/JoinColony/colonyCDapp/issues/2054)

### CODE REVIEW

**Review by melyndav — CHANGES_REQUESTED** [review]
Hey @CzarekDryl thanks for picking up my issue! Appreciate it. :)

Just one bit of feedback as all the rest look to have been resolved. 

1. Can we make the padding inside the white area around the alert card 24px to match Figma. Currently you can see the top and bottom padding is uneven:
image
 
As per Figma:
 
image

This should always be even when a description is shown or not.

**Review by ArmandoGraterol — CHANGES_REQUESTED** [review]
Looking good to me! We just need to fix the lint error related to the class name order

File: src/components/v5/shared/UserPopover/partials/UserInfo.tsx [hunk 1]
```diff
@@ -65,15 +65,14 @@ const UserInfo: FC<UserInfoProps> = ({
               text={formatText({ id: 'userInfo.top.contributor.in' })}
             />
             <div className="flex gap-1">
-              {domains
-                ?.slice(0, 3)
-                .map(({ domainName, domainId }) => (
+              {domains?.slice(0, 3).map(({ domainName, domainId }) => (
+                <div key={domainId}>
```
Comment (bassgeta) [inline src/components/v5/shared/UserPopover/partials/UserInfo.tsx:69]: any particular reason we wrap this in a div now?

**Review by bassgeta — CHANGES_REQUESTED** [review]
Looks good other than the minor comment I dropped, nice one!

File: src/components/v5/shared/UserPopover/partials/UserInfo.tsx (hunk 1 above)
Comment (CzarekDryl) [reply src/components/v5/shared/UserPopover/partials/UserInfo.tsx]: I had a ts error, but now it's gone, so I'm reverting the old changes.

**Review by iamsamgibbs — DISMISSED** [review]
Looks good to me!

**Review by bassgeta — DISMISSED** [review]
Nice one!

**Review by ArmandoGraterol — COMMENTED** [review]
I think there might be too much padding now:
Screenshot 2024-03-27 at 12 53 53

It is combining the margin of the top section plus the top padding of the warning

**Review by bassgeta — APPROVED** [review]

**Comment by CzarekDryl** [comment]
@ArmandoGraterol, @iamsamgibbs Could you please take a look at this one?

**Review by iamsamgibbs — CHANGES_REQUESTED** [review]
This is very close now, only one minor issue I noticed.

There is not enough padding above "about" - seems to be when a user is not verified:

Screenshot 2024-04-03 at 14 14 05

Compared to a verified user:
Screenshot 2024-04-03 at 14 14 35

**Review by ArmandoGraterol — COMMENTED** [review]
Nice work! The issue I pointed out last time seems to be fixed. Only 1 thing left and that would be what Sam already metioned

**Comment by bassgeta** [comment]
Can we check if this PR is applicable at all, since #2066 got merged and that took care of some paddings and margins on this component?

**Pull request closed without merging**